week-3/Homework/mandatory/1-oxygen-levels.js

```diff
@@ -9,10 +9,21 @@ To be safe, they need to land on the first unamed planet that has Oxygen levels
 Write a function that finds the oxygen level of the first safe planet - Oxygen between 19.5% and 23.5%
 */
 
-function safeLevels() {
-
+function safeLevels(oxygenLevels) {
+  /*loops through the array using map return converted float array */
+  const floatConversion = oxygenLevels.map(element=>parseFloat(element));//used arrow fuction as map argument.
+  
+  /*finds oxygen level of the floated array it is important to note '.find returns first element that meets the requirement'*/
+  const findOxyLevel = floatConversion.find(element => element > 19.5 && element < 23.5); //used arrow fuction as find argument.
+   
+  /*Converts the returned required oxygen to string*/ 
```

Comment:
This works well, but turning it into a string is slightly more complicated than it could be :) There are a couple of things here:
1. String interpolation will implicitly convert things to strings, so you don't need to call `toString()` (even though you _can_ - it's not broken to, just more code)
2. We tend to _either_ use interpolation _or_ concatenation, but not both.
So you could either write this:
Using interpolation: 
```javascript
const percentify = `${findOxyLevel}%`;
```
or using concatenation:
```javascript
const percentify = findOxyLevel + '%';
```

Like I said - your code works, and is correct, these are just some suggestions for how to make it a little simpler :)